content/en/docs/Responses/Child.md

```diff
@@ -11,30 +11,95 @@ description: >
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
 {
-  "id": "082f435a363c32c57d5edb6a678a28d4",
-  "parent": "e8a0685e3f3ec6f251649af2b58b8617",
-  "isDir": false,
-  "title": "\"polar expedition\"",
-  "album": "Live at The Casbah - 2005-04-29",
-  "artist": "The New Deal",
-  "track": 4,
-  "year": 2005,
-  "coverArt": "mf-082f435a363c32c57d5edb6a678a28d4_6410b3ce",
-  "size": 19866778,
-  "contentType": "audio/flac",
-  "suffix": "flac",
-  "starred": "2023-03-27T09:45:27Z",
-  "duration": 178,
-  "bitRate": 880,
-  "path": "The New Deal/Live at The Casbah - 2005-04-29/04 - \"polar expedition\".flac",
-  "playCount": 8,
-  "played": "2023-03-26T22:27:46Z",
-  "discNumber": 1,
-  "created": "2023-03-14T17:51:22.112827504Z",
-  "albumId": "e8a0685e3f3ec6f251649af2b58b8617",
-  "artistId": "97e0398acf63f9fb930d7d4ce209a52b",
-  "type": "music",
-  "isVideo": false
+    "id": "082f435a363c32c57d5edb6a678a28d4",
+    "parent": "e8a0685e3f3ec6f251649af2b58b8617",
+    "isDir": false,
+    "title": "\"polar expedition\"",
+    "album": "Live at The Casbah - 2005-04-29",
+    "artist": "The New Deal",
+    "track": 4,
+    "year": 2005,
+    "coverArt": "mf-082f435a363c32c57d5edb6a678a28d4_6410b3ce",
+    "size": 19866778,
+    "contentType": "audio/flac",
+    "suffix": "flac",
+    "starred": "2023-03-27T09:45:27Z",
+    "duration": 178,
+    "bitRate": 880,
+    "path": "The New Deal/Live at The Casbah - 2005-04-29/04 - \"polar expedition\".flac",
+    "playCount": 8,
+    "played": "2023-03-26T22:27:46Z",
+    "discNumber": 1,
+    "created": "2023-03-14T17:51:22.112827504Z",
+    "albumId": "e8a0685e3f3ec6f251649af2b58b8617",
+    "artistId": "97e0398acf63f9fb930d7d4ce209a52b",
+    "type": "music",
+    "isVideo": false,
+    "bpm": 134,
+    "comment": "This is a song comment",
+    "sortName": "Polar expedition",
+    "musicBrainzId": "189002e7-3285-4e2e-92a3-7f6c30d407a2",
+    "genres": [
+        "Hip-Hop",
+        "East coast"
+    ],
+    "artists": [
+        {
+            "id": "ar-1",
+            "name": "Artist 1"
+        },
+        {
+            "id": "ar-2",
+            "name": "Artist 2"
+        }
+    ],
+    "displayArtist": "Artist 1 feat. Artist 2",
```

Comment:
in the case of a Child as we are here, how does displayArtist different from artist? "The New Deal" in this case

Reply:
Because this is a different value with a different meaning and it prevent any compatibility issue for server that in the V1 API decided to only return the first artist and ignore the others.

Reply:
ok, so `artist` is the name of the entity that `artistId` points to, where as `displayArtist` is more true to what the user had in their tags

Reply:
Well it most importantly officially handle multiple artists. When original API only supported 1 and some servers did take only the first. The goal is to not touch legacy fields unless necessary to fix a bug.